docs/source/conf.py

```diff
@@ -173,7 +174,10 @@
 
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'https://docs.python.org/': None}
+intersphinx_mapping = {
+    'python': ('https://docs.python.org/', None),
+    'scipy': ('https://docs.scipy.org/doc/scipy/', None),
```

Comment:
Should we add NumPy or CuPy?

Reply:
Yeah, I think they will be good to have even if they are not currently used. I added those two and scikit-image.